web/pages/notifications.tsx

```diff
@@ -112,6 +182,44 @@ function NotificationsContent(props: {
     useGroupedNonBalanceChangeNotifications(privateUser.id)
   const balanceChangeGroupedNotifications =
     useGroupedBalanceChangeNotifications(privateUser.id)
+  const isAdmin = useAdmin()
+
+  const tabs = [
+    {
+      title: 'Notifications',
+      content: (
+        <NotificationsList
+          privateUser={privateUser}
+          groupedNotifications={groupedNotifications}
+          mostRecentNotification={mostRecentNotification}
+        />
+      ),
+    },
+    {
+      title: 'Balance Changes',
+      content: (
+        <NotificationsList
+          groupedNotifications={balanceChangeGroupedNotifications}
+        />
+      ),
+    },
+    {
+      title: 'Settings',
+      content: (
+        <NotificationSettings
+          navigateToSection={section}
+          privateUser={privateUser}
+        />
+      ),
+    },
+  ]
+
+  if (isAdmin) {
```

Comment:
you can use `buildArray()` for this, checking if `isAdmin && {
      title: 'Send Notif',
      content: <SendCustomNotification />,
    }` in the array